OpenSim/Simulation/Model/Model.h

```diff
@@ -266,6 +266,14 @@ OpenSim_OBJECT_NONTEMPLATE_DEFS(Model, ModelComponent);
     **/
     explicit Model(const std::string& filename) SWIG_DECLARE_EXCEPTION;
 
+    /** Satisfy all connections (Sockets and Inputs) in the model, using this
+     * model as the root Component. This is a convenience form of
+     * Component::finalizeConnections() that uses this model as root.
+     */
+    void finalizeConnections() { finalizeConnections(*this); }
+    // Allow overloading.
+   using Component::finalizeConnections;
+
```

Comment:
@chrisdembia Can you clarify why is this necessary if Model is already a Component?

Reply:
Overloading a virtual function causes the base class overloads to be hidden, unless you use `using ...`. https://stackoverflow.com/questions/6727087/c-virtual-function-being-hidden